added 'showTwoMonth' parameter

Eventpicker opened the pull request:
i had a problem specifically on mobile applications. the date picker with 2 months displayed went over the whole screen and was difficult to use. so i introduced a new parameter that only works with datepicker. 'showTwoMonth' is a boolean and is true by default. you can set it to false and only 1 month will be displayed.

if timepicker is activated this parameter has no effect

Eventpicker committed:
added 'showTwoMonth' parameter

enable/disable if one or two month are displayed at custom daterangepicker

Eventpicker committed:
'showTwoMonth' only for datePicker

'showTwoMonth' should only be allowed for datePicker

mikeluyties commented:
This is a great addition. I also have this issue on mobile and want to limit it to a single calendar, but still with the date range functionality.

Eventpicker committed:
"hideRangesWhenCustomFocus" Feature

hides all predefined ranges when the user selects custom range button
default: false

Eventpicker committed:
UI-Fix: Chosen label was wrong

Eventpicker committed:
Bug Fix: hideRangeWhenCustomFocus init

Bug Fix: hideRangeWhenCustomFocus was not made at initialization

kadekdodikwirawan commented:
I was able to create a single calendar while still keeping the date range functionality by adding:  **linkedCalendars: false**

```js
new DateRangePicker(dateRangePicker, {
    startDate: new Date(),
    endDate: new Date(),
    parentEl: dateRangePicker.parentNode,
    linkedCalendars: false
});
```

And in the SCSS:

```scss
.daterangepicker {
    .drp-calendar.right {
        display: none !important;
    }
}
```

Eventpicker commented:
The problem with this solution is that you lose the “next” button.
The "next" button is displayed with the “showTowMonth” parameter when this is set to true or false.

Eventpicker committed:
add: 'before.show.daterangepicker' Trigger

Adding the 'before.show.daterangepicker' Trigger to change params like 'minDate' before rendering the calendar elements

kadekdodikwirawan commented:
> The problem with your solution is that you lose the “next” button. The "next" button is displayed with the “showTwoMonth” parameter when this is set to true or false.

The next button is still apear, just make sure to set ``` linkedCalendars: false ```
![image](https://github.com/user-attachments/assets/9db8cae1-798e-4562-a080-ae9effd3d91b)

Eventpicker committed:
dependency change from moment to dayjs

Eventpicker committed:
bug-fix: Inputs sets no values

Eventpicker committed:
bug-fix: Timepicker increment was not possible
